### DISCUSSION TITLE
When git_placement set signcolumn, its icons conflict with diagnostics icon?

### DISCUSSION THREAD

**Discussion by linrongbin16** [nvim-tree/nvim-tree.lua#1934] [Q&A] [answered]
Hi, I have config git_placement=signcolumn, and enable diagnostics. I dont see the diagnostics icons. Seems they are conflict and diagnostics icon disappear?

One more question, does we have a diagnostics_placement config?

**Comment by alex-courtis** [comment, marked as answer]
I'm using git and diagnostics in the sign column, with diagnostics taking priority: https://github.com/alex-courtis/arch/blob/a82926db74315757455b80ceef0652ffb321d1aa/config/nvim/lua/amc/plugins/nvim-tree.lua

Please [raise a bug](https://github.com/nvim-tree/nvim-tree.lua/issues/new?assignees=&labels=bug&template=bug_report.yml) with a minimal configuration that shows the issue.

See #1510 for diagnostics placement. As usual, PRs are always welcome ;)
